src/components/ContentEditable.tsx

```diff
@@ -67,9 +72,44 @@ const ContentEditable = React.memo(({ style, html, disabled, innerRef, ...props
     props.onChange(event)
   }
 
+  /** Copies the selection text and sets a text/em flag in the clipboard data to detect the source is 'em' on paste. */
+  const handleCopy = (event: React.ClipboardEvent) => {
+    event.preventDefault()
+
+    const state = store.getState()
+    const simplePath = simplifyPath(state, state.cursor!)
```

Comment:
ContentEditable is supposed to be a very low-level component, not aware of the store or thought structure. The only state it has access to is `state.editableNonce` and that is clearly an escape hatch. 

Additionally, putting this logic here adds the hidden invariant that this ContentEditable is within the cursor thought. That may be circumstantially true, but is probably not something we want to rely on. We would like to avoid implicit invariants sneaking in.

Try taking custom `onCopy` and `onCut` handlers like `onPaste`. Then this logic can move up to the `Editable` component which has access to the Redux state and is aware of thought structure. You could also add a custom `useOnCopy`/`useOnCut` hook like the existing `useOnPaste`. Both of these approaches would result in better separation of concerns.